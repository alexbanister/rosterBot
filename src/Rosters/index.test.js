import { shallow } from 'enzyme';
import { Rosters } from './index';
import React from 'react';


describe('Rosters', () => {
  it('should always match the snapshot', () => {
    const wrapper = shallow(<Rosters rosters={[]} />);

    expect(wrapper).toMatchSnapshot();
  });
});
