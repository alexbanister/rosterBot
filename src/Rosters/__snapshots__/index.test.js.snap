// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rosters should always match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    rosters={Array []}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h3>
          Your Team Rosters
</h3>,
        Array [],
        <button
          className="addButton"
          onClick={[Function]}
>
          + Add Roster
</button>,
      ],
      "className": "rosters",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Your Team Rosters",
        },
        "ref": null,
        "rendered": "Your Team Rosters",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "+ Add Roster",
          "className": "addButton",
          "onClick": [Function],
        },
        "ref": null,
        "rendered": "+ Add Roster",
        "type": "button",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            Your Team Rosters
</h3>,
          Array [],
          <button
            className="addButton"
            onClick={[Function]}
>
            + Add Roster
</button>,
        ],
        "className": "rosters",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Your Team Rosters",
          },
          "ref": null,
          "rendered": "Your Team Rosters",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "+ Add Roster",
            "className": "addButton",
            "onClick": [Function],
          },
          "ref": null,
          "rendered": "+ Add Roster",
          "type": "button",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
